docs/section5/Rule5-4.md

```diff
@@ -0,0 +1,34 @@
+
+# Envelope - Rule 5-4  
+
+**Rule ID:** 5-4  
+**Rule Description:** Baseline roof assemblies must conform with assemblies detailed in Appendix A.  
+**Appendix G Section:** Section G3.1-5(b) Building Envelope Modeling Requirements for the Baseline building  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:** None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+
+## Rule Logic:  
```

Comment:
@xingcx for each RDS that uses a function, could we include a section right under Data Lookup than mentions the functions being used and if possible, also include a link to the same.
This RDS uses 2 function - determining zone space conditioning category and determine opaque surface type. It would help to just list them in the beginning so its clear.

Reply:
Updated with function calls. zone_space_conditioning_category function is not directly cited in the RDS (It is used in the surface_conditioning_category function). Do we want to list it here? @supriyagoel

Reply:
@xingcx if a function is not cited in the RDS then we need not list it here. You're right that the Surrface Conditiong Catergory function calls this function, so it need not be listed here as well.